README.md

```diff
@@ -7,6 +7,10 @@ the context of a project at EPFL.
 [![Test Coverage](https://api.codeclimate.com/v1/badges/97269ca5f086f0c8ed40/test_coverage)](https://codeclimate.com/github/brunnerant/sdp-project/test_coverage)
 [![Maintainability](https://api.codeclimate.com/v1/badges/97269ca5f086f0c8ed40/maintainability)](https://codeclimate.com/github/brunnerant/sdp-project/maintainability)
 
+## Wiki
+We have a Wiki were we keep track of some of our design choices, and were we collect some useful information about android
```

Comment:
It should be `where` instead of `were`

Reply:
Thank you for your sharp eye ;), I will fix it.